alpha/apps/kaltura/lib/model/objectfilters/UserEntryFilter.php

```diff
@@ -28,6 +28,7 @@ protected function init()
 			"_eq_extended_status",
 			"_in_extended_status",
 			"_notin_extended_status",
+			"_bitor_extended_status",
```

Comment:
Why is it UserEntryFilter.php it should be on a new class called PermissionLevelUserEntryFilter that extends this class.